_source/reference/events.md

```diff
@@ -37,3 +37,5 @@ Turbo emits events that allow you to track the navigation lifecycle and respond
 * `turbo:frame-render` fires right after `<turbo-frame>` element renders its view. The specific `<turbo-frame>` element is the event target. Access the `FetchResponse` object with `event.detail.fetchResponse` property.
 
 * `turbo:frame-load` fires when `<turbo-frame>` element is navigated and finishes loading (fires after `turbo:frame-render`). The specific `<turbo-frame>` element is the event target.
+
+* `turbo:fetch-request-error` fires when a form or frame fetch request fails due network errors.
```

Comment:
```suggestion
* `turbo:fetch-request-error` fires when a form or frame fetch request fails due network errors. The specific form or frame element is the event target, respectively.
```

Reply:
Thanks. Updated now to use the same description as other similar events in the document.